src/main/resources/com/mathworks/ci/UseMatlabVersionBuildWrapper/help.html

```diff
@@ -1,3 +1,9 @@
-<p>
-Specify the MATLAB version for this build.
-</p>
\ No newline at end of file
+<div>
+    <p>
+        Use this list to specify the MATLAB version for the build. The plugin prepends your specified MATLAB to the PATH environment variable and uses it to perform the build. <br>
+        The list contains the names of all MATLAB versions that have been registered as a tool in <u>Jenkins Global Tool Configuration</u>. Select one of the tools, or select <b>Custom…</b> if you want to specify a different version. When you select <b>Custom…</b>, you must enter the full path to the desired MATLAB root folder in the <b>MATLAB root</b> box.
+    </p>
+    <p><br>
+        <b>Note:</b> For a Multi-configuration project that has 'MATLAB' axis added, 'Use MATLAB Version' value takes precedence over the values specified by the 'MATLAB' axis.
+    </p>

```

Comment:
Not sure if this note is appropriate for free-style projects.

Some edits: 

<b>Note: </b>In multi-configuration projects, the values specified by <b>Use MATLAB version</b> take precedence over the values specified by the <b>MATLAB</b> axis.

Reply:
What is the current help text for the "MATLAB root" box?

Reply:
![gbl_tool_helptext](https://user-images.githubusercontent.com/68549146/103334047-17379300-4a96-11eb-9fd3-f82e08cc3d52.PNG)

This is what both the help text look like.

Reply:
Applied the suggested edits.

Reply:
A few comments on the help text for MATLAB root:

1) Now that we have shipped 20b, it makes sense to change R2019a to R2020b.

2) Change Mac to macOS. 

3) Rewrite the second sentence to be in active voice:
The plugin prepends the folder to the PATH environment variable and uses the specified MATLAB version to perform the build.

3) Rewrite the rest of the paragraph as follows:
You must not leave this box blank if no MATLAB folder exists on the path. If you leave it blank, the plugin attempts to use the topmost MATLAB folder on the system path.

5) The word "Example" alone looks weird. Probably you can replace it with something like this:
Here are examples of the root folder path on different platforms: